src/layouting/boundingbox.jl

```diff
@@ -13,11 +13,11 @@ the `model` matrix. Plots with `exclude(plot) == true` are excluded.
 
 See also: [`data_limits`](@ref)
 """
-function boundingbox(scenelike, exclude = (p)-> false)
+function boundingbox(scenelike, exclude = (p)-> false, space = :world)
```

Comment:
The order of arguments then has to change so space is first, no?  Either that or it becomes a kwarg...

Reply:
The pr is only really supposed to change `boundingbox(::Text)` to include space so we don't have to deprecate it again in the future. So if this is problematic we can just remove it for now and deal with it later